web_app/web_app.js

```diff
@@ -46,29 +46,38 @@ app.use (compression ())
 app.use (express.static (path.resolve (__dirname, './client/static'), { 'index': ['index.html'] } ));
 app.use (express.static (path.resolve (__dirname, './client/dist')));
 
-/* API to retrieve the 100 most recent pages */
-app.get ('/Pages/', function onListenEvent (req, res) {
-    let statement = db.prepare ('select * from pages order by rx_date desc, recipient asc limit 150');
-    statement.all ([], (error, rows) => {
-        if (error) {
-            throw error;
-        }
-        fix_dates (rows);
-        res.send (rows);
+/* A small wrapper around a app.get handler.
+   This abstracts away generic code that is used on all API requests. */
+function GET(url, handler) {
```

Comment:
The SQLite library doesn't support the async/await pattern, so this is a fair bit uglier than the postgres stuff.